.github/workflows/test.yml

```diff
@@ -23,6 +23,49 @@ jobs:
       - name: Run tests
         run: ./.ci/test.ts
 
+  integration-test:
+    name: Run Integration Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install script dependencies
+        run: npm install --prefix ./.ci
+      - name: Setup dotnet
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: "7.0.x"
```

Comment:
I think you don't need dotnet, now that you don't run dotnet build/restore anymore